packages/lime-cc/files/lime-cc-server.defaults

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+uci set bmx7.sms='plugin'
+uci set bmx7.sms.plugin='bmx7_sms.so'
+
+uci set bmx7.lime_defaults='syncSms'
+uci set bmx7.lime_defaults='lime-defaults'
+
+
```

Comment:
It looks like you like empty lines... I would try to use less empty lines in general for the code